jsapp/js/projects/customViewRoute.tsx

```diff
@@ -111,7 +111,10 @@ function CustomViewRoute() {
 
         {selectedAssets.length === 1 && (
           <div className={styles.actions}>
-            <ProjectQuickActions asset={selectedAssets[0]} />
+            <ProjectQuickActions
+              asset={selectedAssets[0]}
+              isProjectView={true}
```

Comment:
Or we could not use it at all, and instead base the `ProjectQuickActions` functionality on whether `asset` object has `asset.permissions` or not. See the first comment inside the `ProjectQuickActions` function.

Reply:
Or actually we might not need this at all. Looking deeper into this:
- Custom Project View endpoint `/api/v2/project-views/<uid>/assets` lists asset, but without the `permission` object in them (performance reasons)
- Inside the `ProjectQuickActions`, at the top, we check: `const isManagingPossible = userCan('manage_asset', asset)` ← this will be always false for `asset` without `asset.permissions`
- So that check you've added `!isManagingPossible && !isProjectView` is not needed, `!isManagingPossible` should always mean the same thing

Reply:
@magicznyleszek Good call on not needing the prop. I have updated ProjectQuickActions with `const isProjectViewAsset = !("permissions" in asset);`.  I believe we still do need this to determine whether or not to disable the sharing button in the list view, though. For an `Asset`, the button should be disabled if a user can't manage sharing (this is how it currently works on the app). For a ProjectViewAsset, however, the button should not be disabled. So I think removing the prop but still checking whether or not the asset has a permissions key is the best bet here. Let me know if I am missing something.